ublox/CHANGELOG.rst

```diff
@@ -2,8 +2,8 @@
 Changelog for package ublox
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+
 2.3.0 (2022-04-13)
-------------------
```

Comment:
Please don't make these changes; we'll do that when we do a release.

Reply:
Apologies, artifact from rebasing, will fix